module/deleteRow.js

```diff
@@ -0,0 +1,16 @@
+import { Books } from './book.js';
+
+function deleteRow() {
+  const index = this.previousElementSibling.innerHTML;
+  const books = new Books();
+  const booksStored = JSON.parse(localStorage.getItem('books'));
+  books.list = booksStored;
+  books.removeItem(index);
+  localStorage.setItem('books', JSON.stringify(books.list));
+  /* eslint-disable */
+    location.reload();
+    /* eslint-enable */
+}
```

Comment:
- [ ] Good job so far :+1: Kindly use `arrow` functions instead of regular functions :+1:

Reply:
Changed the same.